code/modules/cm_tech/techs/marine/tier4/nuke.dm

```diff
@@ -2,15 +2,15 @@
 
 /datum/tech/nuke
 	name = "Nuclear Device"
-	desc = "Purchase a nuclear device. It's the only way to be sure."
+	desc = "Purchase a nuclear device. It's the only way to sort this out for good."
 	icon_state = "nuke"
 
 	required_points = 5
 
 	tier = /datum/tier/four
 
 	announce_name = "NUCLEAR ARSENAL ACQUIRED"
-	announce_message = "A nuclear device has been authorized and will be delivered to requisitions via ASRS."
+	announce_message = "The deployment of a 'Blockbuster' Large Atomic Fission Demolition Device (LAFDEDE) have been authorized and will be delivered to the Requisitions Department, via ASRS."
```

Comment:
ARES, a military AI, probably wouldn't use nicknames for weaponry, and wouldn't explain the acronyn, should probably just say Large Atomic Fission Demolition Device and drop the "blockbuster" and "LAFDEDE"